doc/man3/PKCS12_SAFEBAG_get0_attrs.pod

```diff
@@ -0,0 +1,44 @@
+=pod
+
+=head1 NAME
+
+PKCS12_SAFEBAG_get0_attrs,PKCS12_get_attr_gen - Retrieve attributes from a PKCS#12 safeBag
+
+=head1 SYNOPSIS
+
+ #include <openssl/pkcs12.h>
+
+ const STACK_OF(X509_ATTRIBUTE) *PKCS12_SAFEBAG_get0_attrs(const PKCS12_SAFEBAG *bag);
+
+ ASN1_TYPE *PKCS12_get_attr_gen(const STACK_OF(X509_ATTRIBUTE) *attrs,
+                                int attr_nid)
+
+=head1 DESCRIPTION
+
+PKCS12_SAFEBAG_get0_attrs() retrieves the stack of B<X509_ATTRIBUTE>s from a PKCS#12 safeBag.
+B<bag> is the B<PKCS12_SAFEBAG> retrieve the attributes from.
```

Comment:
This sentence sounds not very natural...

Reply:
Fixed